### PR TITLE
Fee accordion fix

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- Addresses https://github.com/cowprotocol/cowswap/issues/3995 

https://github.com/cowprotocol/cowswap/assets/31534717/7c6f3357-5668-45df-b394-9b633e19fbd8

### CODE REVIEW

File: apps/cowswap-frontend/src/modules/swap/helpers/getSwapButtonState.ts [hunk 1]
```diff
@@ -110,7 +109,7 @@ export function getSwapButtonState(input: SwapButtonStateParams): SwapButtonStat
     return SwapButtonState.ReadonlyGnosisSafeUser
   }
 
-  if (isQuoteExpired(quote?.fee?.expirationDate) && !swapBlankState && hasInput) {
+  if (isQuoteExpired(quote?.fee?.expirationDate) && trade && !input.inputError) {
```
Comment (alfetopito) [inline apps/cowswap-frontend/src/modules/swap/helpers/getSwapButtonState.ts:112]: This is already included in the release... shouldn't be showing up here 🤔

**Review by alfetopito — APPROVED** [review]
Approved but please remove my commit from develop as that code is already in the release (with a different commit id)

**Review by anxolin — COMMENTED** [review]
I think the base branch is not correct, as you have a commit from Leandro in your PR

**Review by elena-zh — APPROVED** [review]
Great!

File: apps/cowswap-frontend/src/modules/swap/helpers/getSwapButtonState.ts (hunk 1 above)
Comment (shoom3301) [reply apps/cowswap-frontend/src/modules/swap/helpers/getSwapButtonState.ts:112]: I think this is because of changing target branch

Comment (shoom3301) [reply apps/cowswap-frontend/src/modules/swap/helpers/getSwapButtonState.ts]: Fixed

**Pull request merged**